### PR TITLE
Patch internal solution entrypoint files

### PR DESCRIPTION
**Pull request by mikeymike**
When patching a student's given solution, we also need to patch our internal solution.

This ensures that the comparison is like for like and the output solution would match what a student would have to provide.

### CODE REVIEW

File: src/Listener/CodePatchListener.php [hunk 1]
```diff
@@ -36,34 +38,34 @@ public function __construct(CodePatcher $codePatcher)
...
+            $files[] = $exercise->getSolution()->getEntryPoint();
         }
 
-        $this->originalCode = (string) file_get_contents($fileName);
-        file_put_contents(
-            $fileName,
-            $this->codePatcher->patch($event->getExercise(), $this->originalCode)
-        );
+        foreach (array_filter($files) as $fileName) {
+            $this->originalCode[$fileName] = (string) file_get_contents($fileName);
```
Comment (AydinHassan) [inline src/Listener/CodePatchListener.php:49]: Does it work if the files are the same name? I think it’s using full paths right so shouldn’t matter?

Comment (mikeymike) [reply src/Listener/CodePatchListener.php:49]: Comes in as full paths from what I can see, so technically shouldn't matter 👍

**Comment by mikeymike** [comment]
The CS failure looks a little annoying tbh 😂

**Comment by AydinHassan** [comment]
That is annoying, you can probably just ignore it in the config file if we have one

**Comment by mikeymike** [comment]
> That is annoying, you can probably just ignore it in the config file if we have one

I thought so too at first but we don't have the config setup atm, feels too much to add just for this. In all honesty, it just messes with the mock `with` matcher if I change it to valid code so I could just fixup that

**Comment by mikeymike** [comment]
Hopefully fixed CS issue 👍

**Pull request merged**